cmd/fleet/cron.go

```diff
@@ -262,7 +263,7 @@ func cronVulnerabilities(
 
 				default:
 					level.Error(logger).Log("err", "attempting to process vuln automations", "details", "no automation enabled")
-					sentry.CaptureException(err)
+					sentry.CaptureException(fmt.Errorf("attempting to process vuln automations: no automation enabled"))
```

Comment:
`errors.New` :) (doesn't have arguments)

Reply:
Ah thank you! I think you just answered my question above?

Reply:
Ah yes, `sentry.CaptureException(err)` expects err to be an `error` whereas `Logger.Log()` is more generic and supports strings, errors, etc. (`interface{}`).

Reply:
This will fail `go vet ./cmd/fleet/...` with:
```
go vet ./cmd/fleet/...
[...] Errorf format %s reads arg #1, but call has 0 args
```
`errors.New` and `fmt.Errorf` both return an `error`, the former is used to create an `error` from a single string (no arguments), latter is used to create an `error` with a string and arguments.

PS: Go's convention is to suffix methods that take `(s string, args ...interface{})` with an `f`, e.g. `fmt.Errorf`.